src/components/WindowCanvasNavigationControls.js

```diff
@@ -22,35 +23,39 @@ export class WindowCanvasNavigationControls extends Component {
   /** */
   render() {
     const {
-      visible, window, zoomToWorld,
+      classes, visible, window, zoomToWorld,
     } = this.props;
 
     if (!visible) return (<></>);
 
     return (
-      <div
-        className={classNames(ns('canvas-nav'), this.canvasNavControlsAreStacked() ? ns('canvas-nav-stacked') : null)}
-      >
-        <ZoomControls
-          displayDivider={!this.canvasNavControlsAreStacked()}
-          windowId={window.id}
-          zoomToWorld={zoomToWorld}
-        />
-        <ViewerNavigation window={window} />
-        <ViewerInfo windowId={window.id} />
-      </div>
+      <Paper square className={classes.controls} elevation={0}>
```

Comment:
Do we still need the `div` too, or can those classes go onto `Paper`?

Reply:
Nice catch. I think we can 🔥 .. pushed up